api/v1/db/oracledb/member-dao.js

```diff
@@ -96,9 +97,64 @@ const validateMembers = async (memberIds) => {
   }
 };
 
+
+/**
+ * @summary Post a new member into the system.
+ * @function
+ * @param {Object} body The post body from request object.
+ * @returns {Object} The JSON resource of the new member created.
```

Comment:
I think the return type should be `Promise<Object>` since the function is marked `async`